purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/CountdownComponent.kt

```diff
@@ -0,0 +1,34 @@
+package com.revenuecat.purchases.paywalls.components
+
+import androidx.compose.runtime.Immutable
+import com.revenuecat.purchases.InternalRevenueCatAPI
+import com.revenuecat.purchases.utils.serializers.ISO8601DateSerializer
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.Date
+
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+@SerialName("countdown")
+@Immutable
+class CountdownComponent(
+    @get:JvmSynthetic val style: CountdownStyle,
```

Comment:
Hmm I was wondering if we would want to create a `PartialCountdownComponent` to support overrides, the same way we do in other components... But maybe it's not needed for now? We would only be overriding the date right now, which feels unnecessary IMO. So I think we're good 😅

Reply:
I believe I asked this to @joshdholtz and he mentioned we can skip it for now. But let's confirm

Reply:
Confirmed with @joshdholtz we don't need this. Countdown is made of Stacks and Stacks have overrides so it's not needed for countdown